dotnet/samples/KernelSyntaxExamples/README.md

```diff
@@ -59,6 +59,7 @@ dotnet user-secrets set "KeyVault:TenantId" "..."
 
 dotnet user-secrets set "HuggingFace:ApiKey" "..."
 dotnet user-secrets set "HuggingFace:ModelId" "..."
+dotnet user-secrets set "HuggingFace:EmbeddingModelId" "..."
```

Comment:
Given there are so many HF models and discovering the ones that support embeddings is not obvious can you provide a default here?